docs/3.0.0-beta.x/community/contributing-guide.md

```diff
@@ -0,0 +1,75 @@
+# Open Development & Community Driven
+
+Strapi is an open-source project administered by [the Strapi team](https://strapi.io/company). We appreciate your interest and efforts in contributing to Strapi.
+
+There are many ways that you can contribute to Strapi. Contributions to Strapi and the greater Strapi Community can take the form of answering questions in one of the [many Strapi channels](/3.0.0-beta.x/community/contributing-guide.html#get-in-touch), writing articles, creating videos and otherwise publishing self-generated content.
```

Comment:
`greater Strapi community`?

Reply:
This means that people are not only contributing to the Strapi Software itself but are also helping the entire community.  "Greater" means all including.  You may hear it like "the Greater Paris Region" which would include Neuilly or Levallois Perret etc.. 

So in this case, it's not only Strapi or the Strapi team but the greater Strapi community which includes the Strapi Team as a small part of it

Reply:
It's a common American saying used to describe many small parts joined as one.

Another example is when you see Metro used when discussing a large city. Such as the Phoenix Metro area has about 20 cities. Same concept.

In this case you have Strapi team, slack community, stack overflow, other language groups (I know of at least a Spanish speaking one for sure) ect. It's a good use of it

Reply:
Thanks I did learn something!